src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1111,100 +1104,96 @@ window.qBittorrent.DynamicTable = (function() {
                 const date = new Date(this.getRowValue(row) * 1000).toLocaleString();
-                td.set('html', date);
+                td.set('text', date);
                 td.set('title', date);
             };
 
             // completion_on
             this.columns['completion_on'].updateTd = function(td, row) {
                 const val = this.getRowValue(row);
                 if ((val === 0xffffffff) || (val < 0)) {
-                    td.set('html', '');
+                    td.set('text', '');
                     td.set('title', '');
                 }
                 else {
                     const date = new Date(this.getRowValue(row) * 1000).toLocaleString();
-                    td.set('html', date);
+                    td.set('text', date);
                     td.set('title', date);
                 }
             };
 
-            // seen_complete
-            this.columns['seen_complete'].updateTd = this.columns['completion_on'].updateTd;
-
             //  dl_limit, up_limit
             this.columns['dl_limit'].updateTd = function(td, row) {
                 const speed = this.getRowValue(row);
                 if (speed === 0) {
-                    td.set('html', '∞');
+                    td.set('text', '∞');
                     td.set('title', '∞');
                 }
                 else {
                     const formattedSpeed = window.qBittorrent.Misc.friendlyUnit(speed, true);
-                    td.set('html', formattedSpeed);
+                    td.set('text', formattedSpeed);
                     td.set('title', formattedSpeed);
                 }
             };
 
             this.columns['up_limit'].updateTd = this.columns['dl_limit'].updateTd;
 
-            // downloaded, uploaded, downloaded_session, uploaded_session, amount_left, completed, total_size
+            // downloaded, uploaded, downloaded_session, uploaded_session, amount_left
             this.columns['downloaded'].updateTd = this.columns['size'].updateTd;
             this.columns['uploaded'].updateTd = this.columns['size'].updateTd;
             this.columns['downloaded_session'].updateTd = this.columns['size'].updateTd;
             this.columns['uploaded_session'].updateTd = this.columns['size'].updateTd;
             this.columns['amount_left'].updateTd = this.columns['size'].updateTd;
-            this.columns['amount_left'].updateTd = this.columns['size'].updateTd;
-            this.columns['completed'].updateTd = this.columns['size'].updateTd;
-            this.columns['total_size'].updateTd = this.columns['size'].updateTd;
 
-            // save_path, tracker
-            this.columns['save_path'].updateTd = this.columns['name'].updateTd;
-            this.columns['tracker'].updateTd = this.columns['name'].updateTd;
+            // time active
+            this.columns['time_active'].updateTd = function(td, row) {
+                const time = friendlyDuration(this.getRowValue(row));
```

Comment:
Got an error here, you should use `window.qBittorrent.Misc.friendlyDuration`

Reply:
fixed